src/modules/vendor/components/programs/ProfileContent.vue

```diff
@@ -13,7 +13,7 @@
         @blur="updateSubProgramName(index)" :error="$v.program.subPrograms.$each[index].name.$error"
         :disable="isPublished(subProgram)" />
       <draggable @end="dropStep(subProgram._id)" v-model="subProgram.steps"
```

Comment:
On peut changer le @end en @change, il me semble pour draggable.
Je ne sais pas si tu as déjà review la PR de félix mais il a remplacé @end pour @change pour éviter d'avoir une notification lorsqu'on drag and drop sans faire de changement

Reply:
c'esst changé dans la PR de felix